scripts/zones/Port_San_dOria/npcs/Regine.lua

```diff
@@ -51,32 +53,22 @@ end;
 
 function onEventFinish(player,csid,option)
     -- FLYERS FOR REGINE
-    if (csid == 601) then
-        player:setCharVar("FFR",1);
-    elseif (csid == 510 and option == 2) then
-        if (npcUtil.giveItem(player, {{532,12}, {532,3}} )) then
-            player:addQuest(SANDORIA,tpz.quest.id.sandoria.FLYERS_FOR_REGINE);
-            player:setCharVar("FFR",17);
-        end
-    elseif (csid == 603) then
-        if (npcUtil.completeQuest(player, SANDORIA, tpz.quest.id.sandoria.FLYERS_FOR_REGINE, {gil=440, title=tpz.title.ADVERTISING_EXECUTIVE})) then
-            player:setCharVar("tradeAnswald",0);
-            player:setCharVar("tradePrietta",0);
-            player:setCharVar("tradeMiene",0);
-            player:setCharVar("tradePortaure",0);
-            player:setCharVar("tradeAuvare",0);
-            player:setCharVar("tradeGuilberdrier",0);
-            player:setCharVar("tradeVilion",0);
-            player:setCharVar("tradeCapiria",0);
-            player:setCharVar("tradeBoncort",0);
-            player:setCharVar("tradeCoullene",0);
-            player:setCharVar("tradeLeuveret",0);
-            player:setCharVar("tradeBlendare",0);
-            player:setCharVar("tradeMaugie",0);
-            player:setCharVar("tradeAdaunel",0);
-            player:setCharVar("tradeRosel",0);
-            player:setCharVar("FFR",0);
+    if csid == 601 then
+        player:setCharVar("FFR", 1)
```

Comment:
I know this was always here, but do we happen to know if the pre-quest CS only happens once _ever_, or just once per zone?

Reply:
I'm working off a video -- no retail access -- so can't answer this off the top of my head.  I added this question to an already-open Flyers for Regine question I have in the FFXICaptures discord.